com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Requests.java

```diff
@@ -372,6 +380,7 @@ public static RunInTerminalRequestArguments createExternalTerminal(String[] cmds
         CONTINUEOTHERS("continueOthers", ThreadOperationArguments.class),
         PAUSEALL("pauseAll", ThreadOperationArguments.class),
         PAUSEOTHERS("pauseOthers", ThreadOperationArguments.class),
+        REFRESHVARIABLES("refreshVariables", RefreshVariablesArguments.class),
```

Comment:
`REFRESH_VARIABLES` would be better human-readable. For consistency just leave it as is. But consider to refactor all the  enums one day?

Reply:
yes, i will use a separate PR to refactor these enums.